app/views/layouts/application.html.erb

```diff
@@ -33,7 +33,7 @@
 
       <!-- Notifications -->
       <% flash.each do |type, msg| %>
-        <div class="<%= type %>"><%= msg %></div>
+        <div class="<%= type %>"><%= msg.html_safe %></div>
```

Comment:
Perhaps I'm being paranoid, but wonder if we should run this through something like https://github.com/rgrove/sanitize first

Reply:
fair, I used the built in SanitizeHelper instead